src/main/java/com/baidu/hugegraph/rest/RestClient.java

```diff
@@ -90,9 +93,68 @@ public RestClient(String url, String user, String password, int timeout,
                                      .config(maxTotal, maxPerRoute)
                                      .build());
     }
-    
+
+    public RestClient(String url, String user, String password, int timeout,
+                      int maxTotal, int maxPerRoute, String protocol,
+                      String trustStoreFile, String trustStorePassword) {
+        this(url, new ConfigBuilder().config(timeout)
+                                     .config(user, password)
+                                     .config(maxTotal, maxPerRoute)
+                                     .config(protocol, trustStoreFile, trustStorePassword)
+                                     .build());
+    }
+
+    private TrustManager[] NoCheckTrustManager() {
```

Comment:
method name must be started with lower case, rename to createNoCheckTrustManager